core/src/main/java/org/apache/spark/api/java/StorageLevels.java

```diff
@@ -40,19 +40,6 @@
    * Create a new StorageLevel object.
    * @param useDisk saved to disk, if true
    * @param useMemory saved to memory, if true
-   * @param deserialized saved as deserialized objects, if true
-   * @param replication replication factor
-   */
-  @Deprecated
-  public static StorageLevel create(boolean useDisk, boolean useMemory, boolean deserialized,
-      int replication) {
-    return StorageLevel.apply(useDisk, useMemory, false, deserialized, replication);
-  }
-
-  /**
-   * Create a new StorageLevel object.
-   * @param useDisk saved to disk, if true
-   * @param useMemory saved to memory, if true
    * @param useOffHeap saved to Tachyon, if true
```

Comment:
@srowen sure. It seems that references to Tachyon still exist in docs, code comments, test suits, text resources (here https://github.com/apache/spark/blob/master/core/src/test/resources/spark-events/local-1422981759269 for example), so I'll look around, remove them all, and run thorough tests. Do you mind if I do it in a separate PR?